README.md

```diff
@@ -136,4 +136,99 @@ Push to origin master
     limitations under the License.
 
 ## Contribute
-See [CONTRIBUTING.md](https://github.com/Kinvey/java-library/blob/master/CONTRIBUTING.md) for details on reporting bugs and making contributions.
\ No newline at end of file
+See [CONTRIBUTING.md](https://github.com/Kinvey/java-library/blob/master/CONTRIBUTING.md) for details on reporting bugs and making contributions.
+
+====================X====================
+
+====================X====================
+
+====================X====================
+
+<p align="left">
+  <a href="https://www.progress.com/kinvey" style="display: inline-block;">
+    <img src="logo-progresskinvey.png">
+  </a>
+</p>
+
+# [Kinvey Android SDK](https://devcenter.kinvey.com/android)
+
+![badge-jitpack] ![badge-status] ![badge-coverage]
+
+# Overview
+
+[Kinvey](https://www.progress.com/kinvey) is a high-productivity serverless application development platform that provides developers tools to build robust, multi-channel applications utilizing a cloud backend and front-end SDKs. As a platform, Kinvey provides many solutions to common development needs, such as a data store, data integration, single sign-on integration, and file storage. With Kinvey, developers can focus on building what provides value for their app - the user experience (UX) and business logic of the application. This approach increases developer productivity and aims to enable higher quality apps by leveraging Kinvey's pre-built components.
+
+# Features
+
+The Kinvey Android SDK repository represents the package that can be used to develop Android apps on the Kinvey platform. The Kinvey SDK is developed using a mix of Kotlin and Java code, with a gradual transition being made to a completely Kotlin codebase.
+
+## Contents
+
+The following is a high-level overview of the most important projects in the solution:
+
+* `java-api-core` The core of the library. Most of the library functionality is written here. This project contains most of the underlying networking, user management, caching logic. Things that are Android-platform specific are represented as interfaces / abstract classes, and implemented in the `android-lib` library described below.
+* `android-lib`: The wrapper library for Android, built on top of `java-api-core`. All the Android-specific implementation is located here, and most of the classes in this library extend from the ones in `java-api-core`.
+
+## Using the SDK
+
+Refer to the Kinvey [DevCenter](http://devcenter.kinvey.com/android) for guides and documentation on using Kinvey.
+
+# Releases
+
+Versioning of the Kinvey SDK follows the guidelines stated in [Semantic Version 2.0.0](http://semver.org/).
+
+# Build Instructions
+
+In order to build this repository, the following pre-requisites must be in place:
+
+* [android sdk](http://developer.android.com/sdk/index.html)
+* Set JAVA_HOME
+* Set ANDROID_HOME
+* Download android_sdk/platforms/android-19, android_sdk/platforms/android-10
```

Comment:
This text can be removed: `Download android_sdk/platforms/android-19, android_sdk/platforms/android-10`.
Because now we don't use `Doclava` javadoc generation tool that was required direct paths to this SDKs.